src/helpers.php

```diff
@@ -13,6 +13,7 @@
     function current_route(string $locale = null): string
     {
         $route = Route::getCurrentRoute();
+        $parameters = $route->parameters;
         $query = request()->getQueryString();
```

Comment:
We can force an array like this here for the array merge instead: `$query = (array) request()->getQueryString()`. When the `$query` would be null, it would become an empty array.